.eslintrc.json

```diff
@@ -34,6 +34,15 @@
       "error",
       { "components": ["label"], "allowChildren": true }
     ],
+    "jsx-a11y/label-has-associated-control": [
```

Comment:
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md#case-my-label-and-input-components-are-custom-components